### DISCUSSION TITLE
Noob Question

### DISCUSSION THREAD

**Discussion by homfgomery** [ruffle-rs/ruffle#4277] [Q&A]
Hey I'm just starting to learn HTML/CSS/JavaScript and I'm messing around building my own website. Am I just a novice or is it not possible to make it so a flash banner ad pops up with a click of a button using Ruffle? Maybe I'm misunderstanding the use of Ruffle entirely, IDK! 

Even a "yes this is possible" or "no this isn't possible" would be appreciated.

Thanks! 

https://archive.org/details/bannerad_timecapsule_2003-4 
^theres the archive of flash ads im using and trying to run

**Comment by jhwgh1968** [comment]
If [you embed Ruffle on your website](https://ruffle.rs/#usage), I imagine you can do something like that with [the Javascript API](https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#javascript-api).

I suspect the button would have to be Javascript, though.
